tests/unittest_qmail-send.c

```diff
@@ -0,0 +1,254 @@
+#include <stdio.h>
+#define DEPRECATED_FUNCTIONS_REMOVED
+#include <check.h>
+#include "qmail-send-without-main.c"
+
+extern struct job *jo;
+extern int numjobs;
+
+// START test helpers for the API we might prefer from qmail-send
+
+#define available 0
+#define unavailable 1
+
+static void job_init_this_many(int numslots) 
+{
+  numjobs = numslots;
+  job_init();
+}
+
+static void occupy_job(unsigned int index) 
+{
+  jo[index].refs = unavailable;
+}
+
+static void occupy_all_jobs(void) 
+{
+  int j;
+  for (j = 0; j < numjobs; j++) 
```

Comment:
Please add a newline after the variable just for consistency and easier reading.